python/rikai/io.py

```diff
@@ -124,6 +159,8 @@ def open_uri(uri: Union[str, Path], mode="rb") -> BinaryIO:
     elif parsed_uri.scheme in ("http", "https"):
         resp = requests.get(uri)
         return BytesIO(resp.content)
+    elif parsed_uri.scheme == "gs":
+        return _gcsfs().open(uri, mode=mode)
```

Comment:
I think the type signature is wrong? Depending on `mode`, the return value is not always a `BinaryIO`.

Reply:
Hmm, good point.